pkgs/applications/radio/yaac/default.nix

```diff
@@ -0,0 +1,46 @@
+{ fetchzip
+, jre8
+, lib
+, makeWrapper
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  version = "1.0-beta181";
+  pname = "yaac";
+
+  src = fetchzip {
+    url = "https://www.ka2ddo.org/ka2ddo/YAAC.zip";
+    hash = "sha256-15oH+I/ACbcHQ6bMohKoO1+GKDu8ThkjKTKN0JIHty8=";
+    stripRoot = false;
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  # TODO: Make Desktop Icon
```

Comment:
Why not do this now?

Reply:
Not sure of the best way to achieve it. 

There's a 32 and 64 ico file in the images dir in the output, I just haven't had much luck in creating it with the makeDesktopItem/copyDesktopItems hooks in my limited attempts at it. Also not sure of disposing of them when the package is removed, I've had some nix-env app icons installed that persist after uninstall.

The binary created is mostly what's used but a desktop icon is just a nice to have.

Reply:
> Also not sure of disposing of them when the package is removed, I've had some nix-env app icons installed that persist after uninstall.

that's on nix-env maybe leaving symlinks behind, nothing nixpkgs can really fix.



> I just haven't had much luck in creating it with the makeDesktopItem/copyDesktopItems hooks in my limited attempts at it.

I can't really provide you with help if I don't have more details.